src/harrastuspassi/harrastuspassi/management/commands/import_lipas.py

```diff
@@ -95,6 +95,7 @@ def handle(self, *args, **options):
                         continue
                     location, location_created = Location.objects.update_or_create(
                         data_source=self.source,
+                        origin_id=sports_place.get('sportsPlaceId'),
                         name=sports_place.get('name'),
```

Comment:
May be throughout the code everything else but `data_source` and `origin_id` should move in defaults? otherwise, should 'rata 3.5 km' be someday renamed into 'juoksurata 3.5 km' we'll get a duplicate in the db.

Reply:
That is true, the updating conditions should be updated, but it would break since we don't have `origin_id` populated. My suggestion is that we will release this, and change the updating conditions in a later PR.

Reply:
good point! didn't think of the current entries. They need to be removed anyways, for with this PR we'll create duplicates. Same names, but non-empty origin-ids, or am I wrong on that? So if we have shell access to staging and prod, removing all entries with lipas datasource would be the easiest, if not - data migration, doing the same thing.